index.bs

```diff
@@ -3318,8 +3318,7 @@ Create a named {{MLOperand}} based on a descriptor, that can be used as an input
         The permissions and context validity have been checked by [[#api-mlgraphbuilder-constructor]] steps.
     </div>
     1. If |name| is empty, then [=exception/throw=] a {{TypeError}}.
-    1. [=Assert=]: If |descriptor|.{{MLOperandDescriptor/dimensions}} does not [=map/exist=], then |descriptor| defines a scalar input.
-    1. If |descriptor|.{{MLOperandDescriptor/dimensions}} [=map/exists=]:
+    1. If |descriptor|.{{MLOperandDescriptor/dimensions}}'s [=list/size=] is not 0:
         1. If [=checking dimensions=] given |descriptor|.{{MLOperandDescriptor/dataType}} and |descriptor|.{{MLOperandDescriptor/dimensions}} returns false, then [=exception/throw=] a "{{DataError}}" {{DOMException}}.
         1. If |descriptor|'s [=byte length=] is not supported by the underlying platform, then [=exception/throw=] a "{{DataError}}" {{DOMException}}.
```

Comment:
> If |descriptor|'s [=byte length=] is not supported by the underlying platform, then [=exception/throw=] a "{{DataError}}" {{DOMException}}.

I wonder if this line makes more sense to move into "check dimensions" to apply generically, as it already has this closely related bit:

`If ... it is too large to be supported by the implementation given type, return false.`

Reply:
Yeah, and it's there in constant() too...

Reply:
Okay, this was a bit more intrusive so I did it as a separate commit, but I liked where it ended up. Take a look?

Reply:
(uh, last two commits together)